db/seeders/letter_template_seeder.rb

```diff
@@ -4,7 +4,7 @@ def seed!
     puts "---- Seeding Letter Templates ----"
     # Reciever for prision related string
     prison_receiver = <<~EOF
-        <%= letter.name %><% if values.prison_number.present? %> - <%= values.prison_number %><% if (values.prison_number.gsub(/[,]/, ' ').squeeze(' ').strip).match?(/[ ]/) %> [DELETE AS APPROPRIATE]<% end %><% end %>
+        <%= letter.name %><% if values.prison_number.present? %> - <%= values.first_prison_number %><% end %>
```

Comment:
Why do we `upcase` everywhere else other than here? Seems like it would be better to be uniform with this 🤔